python/fastquant/backtest/backtest.py

```diff
@@ -129,12 +108,14 @@ def backtest(
         df of sentiment [0, 1] indexed by time (applicable if `strategy`=='senti')
     strats : dict
         dictionary of strategy parameters (applicable if `strategy`=='multi')
-    data_format : str
-        input data format e.g. ohlcv (default=None so format is automatically inferred)
 
     {0}
     """
 
+    if data_format:
+        print("Warning: data_format argument is no longer needed since formatting is now purely automated based on column names!")
```

Comment:
```python
import warnings #move to top

errmsg = "Warning: data_format argument is no longer needed since formatting is now purely automated based on column names!"
errmsg += "\nWe will be removing this argument in a coming release!"
warnings.warn(errmsg, DeprecationWarning)
print(errmsg)
```

Reply:
Good catch!